ts/client/src/client.ts

```diff
@@ -4339,8 +4339,10 @@ export class MangoClient {
           1) *
         100;
     }
-    const maxPricePremiumRate =
+    let maxPricePremiumRate =
       maxPricePremiumPercent > 0 ? maxPricePremiumPercent / 100 : 0.03;
+    maxPricePremiumRate =
+      maxPricePremiumRate * premiumMultiplier + extraPricePremiumBps / 10000;
```

Comment:
Imo this modification of user-specified premium is weird. Would prefer if this was just done externally to the price premium argument. This function should just build the instruction with the passed-in args.

Reply:
They do have defaults which act as no-op